src/contexts/cart&wishlist/reducer.js

```diff
@@ -13,36 +12,46 @@ export const cartWishlistReducer = (state, { type, payload }) => {
         ...state,
         apiUrl: "/api/user/cart",
         apiMethod: "get",
-        postData: { product: {} },
+        postData: {},
       };
     case "addItemToCart":
       return {
         ...state,
         apiUrl: `/api/user/cart`,
         apiMethod: "post",
         postData: { product: { ...payload, qty: 1 } },
-        toastMessage:`${payload.name} added to cart`,
-        toastType:"success"
+        toastMessage: `${payload.name} added to cart`,
+        toastType: "success",
```

Comment:
Not sure if it is good way to keep toast messages inside reducers.

Reply:
I just did that to avoid creating another reducer for toast messages and since the toasts would be fired only after one of cases in the reduces satisfies, so put the toast states in it